dvc/render/convert.py

```diff
@@ -29,10 +29,8 @@ def _group_by_rev(datapoints):
 
 
 def to_json(renderer, split: bool = False) -> List[Dict]:
```

Comment:
@skshetry just to double check - can it be used in Studio directly by chance?

Reply:
Does not seem so. `rg "to_json"` does not return any hit.